flocx_market/api/app.py

```diff
@@ -1,20 +1,17 @@
-import pecan
-from pecan import make_app
-from flocx_market import model
-from flocx_market.api import config as api_config
+import os
+from flask import Flask
+from flask import Blueprint
+from flask import jsonify
+my_bp = Blueprint("my_bp", __name__)
 
 
-def get_pecan_config():
-    filename = api_config.__file__.replace('.pyc', '.py')   # get the absolute path of the pecan config.py
-    return pecan.configuration.conf_from_file(filename)
+@my_bp.route('/')
+def index():
+    version = {"versions": {"values": [{"status": "in progress", "updated": "2019-07-02T00:00:00Z", "media-types": [{"base": "application/json", "name": "flocx-marketplace"}], "links": [{"href": "http://localhost:8081", "rel": "self"}]}]}}
```

Comment:
This is probably more readable if you put the lines of the dict on different lines. I think we're also calling it 'flocx-market' rather than 'flocx-marketplace'; and instead of hardcoding `http://localhost:8081` we should get it from oslo config.

Reply:
Should I modify default value for CONF.api.public_endpoint and use it to get the flocx-market url? Then I need to modify some places in config test, since the default value is None.

Reply:
The WSGI service starts up using these parameters, right?

```
            host=CONF.api.host_ip,
            port=CONF.api.port,
```

Just use those.

Reply:
Thank you! All fixed.